subprojects/docs/src/docs/userguide/javaPlugin.adoc

```diff
@@ -873,7 +874,7 @@ If you don't want to use test class detection, you can disable it by setting `sc
 [[test_grouping]]
 ==== Test grouping
 
-JUnit and TestNG allows sophisticated groupings of test methods.
+JUnit/JUnit Platform and TestNG allows sophisticated groupings of test methods. Currently, http://junit.org/junit5/docs/current/user-guide/#writing-tests-tagging-and-filtering[JUnit Platform filtering and tagging] is not supported yet.
```

Comment:
I am wondering if we should explicitly call out JUnit 3/4 here instead of just JUnit.